### PR TITLE
Cu 86c0zbfdg dockerise mediator

### PR DESCRIPTION
**Pull request by ItsMurumba**

### CODE REVIEW

File: .github/workflows/docker-publish.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+on:
+  push:
+    tags:
+      - "*.*.*"
+    branches:
+      - "*"
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
```
Comment (drizzentic) [inline .github/workflows/docker-publish.yml:13]:
```suggestion
    environment: dockerhub
    runs-on: ubuntu-latest
```

**Review by drizzentic — DISMISSED** [review]
LGTM! just ensure to format and add new line at end of the files without one.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM node:16
```
Comment (brett-onions) [inline Dockerfile]: Suggestion: maybe use node 16 alpine for smaller image size

**Review by brett-onions — DISMISSED** [review]
Looks good just a minor suggestion

**Review by drizzentic — APPROVED** [review]

**Pull request merged**